checker/manual/external-tools.tex

```diff
@@ -302,17 +302,17 @@ \subsection{Explanation\label{ant-task-explanation}}
 \section{Maven\label{maven}}
```

Comment:
Hi Renato,
I tried your instructions with Guava. We were able to get it work with JDK8 on Guava with the additional instructions I provided in the comments below. Could you please incorporate these additional instructions, and also incorporate these instructions as additional instructions that work with JDK8, rather than instructions that completely replace the original ones, in the way that Mike mentions above?
Thank you,
Javier